README.md

```diff
@@ -78,6 +84,11 @@ Other active channels:
 
 - Gitter Chat: [dragonfly](https://gitter.im/alibaba/Dragonfly)
 - Twitter: [@dragonfly_oss](https://twitter.com/dragonfly_oss)
+- Dingtalk Group(钉钉群)
+
+<p align="center">
+<img src=https://user-images.githubusercontent.com/9465626/50591975-2ac2ba00-0ecd-11e9-9e53-4c4d6a7dfc7d.jpg width="250">
```

Comment:
This part will hit the rule MD034 in markdownlint, please refer to https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md034---bare-url-used